unime/src/i18n/en/index.ts

```diff
@@ -207,7 +207,7 @@ const en = {
     NAVBAR_TITLE: 'Connected',
     TABS: {
       CONNECTIONS: 'Connections',
-      TIMELINE: 'Timeline',
+      HISTORY: 'History',
```

Comment:
Can you update these tags for NL and DE as well? (Same for `HISTORY` below in line 255)